packages/expo/src/winter/url.ts

```diff
@@ -106,7 +106,7 @@ URL.revokeObjectURL = function revokeObjectURL(_url) {
 
 URL.canParse = function canParse(url: string, base?: string): boolean {
   try {
-    URL(url, base);
+    new URL(url, base);
```

Comment:
Looks like we have a (not so helpful) lint rule on this case

> 109:5  warning  Do not use 'new' for side effects  no-new

```suggestion
    // eslint-disable-next-line no-new
    new URL(url, base);
```